projects/robots/bitcraze/crazyflie/protos/Crazyflie.proto

```diff
@@ -30,153 +30,195 @@ PROTO Crazyflie [
     model "Bitcraze's Crazyflie"
     translation 0 0 0.015
     children [
-      DEF BODY Transform {
+        DEF BODY Transform {
         translation 0 0 -0.015
         children [
           Group {
           children IS extensionSlot
-          }
-          CadShape {
+            }
+            CadShape {
             url [
               "meshes/cf2_assembly.dae"
             ]
-          }
+            }
         ]
-      }
-      DEF MULTIRANGER Transform {
+        }
+        DEF MULTIRANGER Transform {
         translation 0 0 0.01
         children [
-          DistanceSensor {
+            DistanceSensor {
             rotation 0 0 1 -1.57
             name "range_right"
             lookupTable [
-              0 0 0
-              2 2000 0
+                0 0 0
+                2 2000 0
             ]
-          }
-          DistanceSensor {
+            }
+            DistanceSensor {
             rotation 0 0 1 3.14
             name "range_back"
             lookupTable [
-              0 0 0
-              2 2000 0
+                0 0 0
+                2 2000 0
             ]
-          }
-          DistanceSensor {
+            }
+            DistanceSensor {
             rotation 0 0 1 1.57
             name "range_left"
             lookupTable [
-              0 0 0
-              2 2000 0
+                0 0 0
+                2 2000 0
             ]
-          }
-          DistanceSensor {
+            }
+            DistanceSensor {
             name "range_front"
             lookupTable [
-              0 0 0
-              2 2000 0
+                0 0 0
+                2 2000 0
             ]
-          }
+            }
         ]
-      }
-      Camera {
+        }
+        Camera {
         translation 0.03 0 0.01
         fieldOfView 0.87
         width 324
         height 324
-      }
-      GPS {
-      }
-      Gyro {
-      }
-      InertialUnit {
-      }
-      DEF M1 Propeller {
+        }
+        GPS {
+        }
+        Gyro {
+        }
+        InertialUnit {
+        }
+        DEF M1 Propeller {
         shaftAxis 0 0 1
         centerOfThrust 0.031 -0.031 0.008
         thrustConstants -4e-05 0
         torqueConstants 2.4e-06 0
+        fastHelixThreshold 50
         device RotationalMotor {
-          name "m1_motor"
-          maxVelocity 600
-          maxTorque 30
+            name "m1_motor"
+            maxVelocity 600
+            maxTorque 30
+        }
+        fastHelix Solid {
+            translation 0.031 -0.031 0.008
+            children [
+            DEF FASTHELIX Shape {
+                appearance PBRAppearance {
+                baseColor 0.1 0.1 0.1
+                baseColorMap ImageTexture {
+                    url [
+                    "textures/fast_helix.png"
```

Comment:
Assuming you re-used this one:
```suggestion
                    "webots://projects/robots/dji/mavic/protos/Mavic2Pro/textures/mavic_fast_helix.png"
```

Reply:
alright let's go for that then. I might make a mental note to fix this in the future as it would be nicer to have the meshes and textures separated from the proto without external dependencies, but it won't be urgent.